readme/next-steps.md

```diff
@@ -19,7 +19,6 @@ Here are tasks that are typically performed at this stage:
 
 * Create and deploy an artifact. See [Deployment workflow](deploy.md).
 
-        # Ensure git.remotes are set in blt/project.yml. You should replace the default Bolt Acquia Cloud repository with your Acquia      Cloud repository and you should add your Github repository
```

Comment:
We still need to let people know that they should add their git repository URL here.